examples/grid_example/public/app.tsx

```diff
@@ -11,7 +11,7 @@ import deepEqual from 'fast-deep-equal';
 import { cloneDeep } from 'lodash';
```

Comment:
there's someweird behavior happening (elements shift slightly when resizing) on a small screen but not sure if it comes from this PR (will recheck on main tomorrow!)

https://github.com/user-attachments/assets/f971690a-604e-4867-a9db-ed1e1a62666c

Reply:
I am unable to repro this 🙇 Maybe one of my other changes somehow fixed this? Hmm.....

![May-15-2025 17-11-40](https://github.com/user-attachments/assets/c32b66a5-d668-42cc-a743-6416c58faace)

Reply:
You fixed it here, just checked 🎉 
520d5399f0ef0ae07e2f3788e26535a951b2be2b